packages/patternfly-4/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -2,28 +2,33 @@
 title: 'Bar'
 section: 'charts'
 typescript: true
-propComponents: ['Chart', 'ChartAxis', 'ChartBar', 'ChartGroup', 'ChartVoronoiContainer']
+propComponents: [
+  'Chart',
+  'ChartAxis',
+  'ChartBar',
+  'ChartGroup',
+  'ChartVoronoiContainer'
+]
 ---
 
 import { Chart, ChartAxis, ChartBar, ChartGroup, ChartThemeColor, ChartVoronoiContainer } from '@patternfly/react-charts';
-import './chart-bar.scss';
 
+## Introduction
 Note: PatternFly React charts live in its own package at [@patternfly/react-charts](https://www.npmjs.com/package/@patternfly/react-charts)!
 
-## Introduction
 PatternFly React charts are based on the [Victory](https://formidable.com/open-source/victory/docs/victory-chart/) chart library, along with additional functionality, custom components, and theming for PatternFly. This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior.
 
 Learn to build a bar chart using a Katacoda tutorial starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color. You'll learn how to use React chart components together to build a consistent user experience.
 
 [Start course](https://katacoda.com/patternfly/courses/charts/bar-chart)
 
 ## Examples
-```js title=Simple-bar-chart-with-right-aligned-legend
+```js title=Right--aligned-legend
 import React from 'react';
 import { Chart, ChartBar } from '@patternfly/react-charts';
 
-<div>
-  <div className="bar-chart-legend-right">
+RightLegend = (
```

Comment:
We have both `RightLegend` and `RightAlignedLegend`, which sound too similar. Could this be `SimpleBarChart`?

Reply:
We could do this, but our current variable names for classes don't do this. Example: `class MultiColorChart extends React.Component {` in ChartThreshold.md. I'm just going to keep shorter variable names for now that match the titles.

Reply:
I'm not following the logic as to what these names must be? We can make the class names match.

If I were to clean up afterward, can I not use BarChartSimple, BarChartMultiColor, BarCharRightLegend, etc.?

Reply:
These can be anything. The only thing they matter for is what people copy/paste.